### ISSUE TITLE
RCS1213: Can't detect usages in stackalloc

### ISSUE THREAD

**Issue by krisztiankocsis** [JosefPihrt/Roslynator#669]
**Product and Version Used**: Roslynator (NuGet) 2.3.0

**Steps to Reproduce**:
```csharp
class Something {
  private const int BufferSize = 64; // This line is falsely reported by RCS1213.

  private string MyProp {
    get {
      Span<char> dataBuffer = stackalloc char[BufferSize];
      ...
    }
  }
}
```

**Actual Behavior**:
Getting reported by RCS1213.

**Expected Behavior**:
Not reported as it is actually used.
